index.html

```diff
@@ -0,0 +1,136 @@
+<script defer src="script.js"></script>
+</head>
+<body>
+  <header>
+    <div class="container-fluid deskto-top-nav">
+      <div class="social-media-div">
+        <ul class="social-media">
+          <li class="">
+            <a href="" class="social-media-item"><i class="fa-brands fa-facebook-f"></i></a>
+          </li>
+          <li class=""><a href="#" class="social-media-item"><i class="fa-brands fa-twitter"></i></a>
+          </li>
+          <li class="">
+            <a href="#" class="social-media-item">English</a>
+          </li>
+          <li class="">
+            <a href="#" class="social-media-item">Mypage</a>
+          </li>
+          <li class="">
+            <a href="#" class="social-media-item">Logout</a>
+          </li>
+        </ul>
+      </div>
+    </div>
+    <nav class="navbar">
+      <div class="logo"><a href="./index.html"><img src="img/foursquare-logo.png" alt="Foursquare Gospel Church Logo"></a></div>
+      <ul class="nav-menu">
+        <li class="nav-item"><a class="nav-link" href="#home">Home</a></li>
+        <li class="nav-item"><a class="nav-link" href="./about.html">About</a></li>
+        <li class="nav-item"><a class="nav-link" href="">Contact</a></li>
+        <li class="nav-item"><a class="nav-link" href="">Register</a></li>
+        <li class="nav-item convention-link"><a class="foursq-btn" href="#"><button>4sq convetion</button></a></li>
+      </ul>
+      <div class="hamburger">
+        <span class="bar"></span>
+        <span class="bar"></span>
+        <span class="bar"></span>
+      </div>
+    </nav>
+    
+  </header>
+  <section class="home">
+  
+    <div class="container">
+      <p class="theme">"Supernatural Encounter"</p>
+      <h1 class="heading">FOURSQUARE ANNUAL <br> CONVENTION 2023 <br> NAMIBIA</span></h1>
+      <h3 class="sub-heading">Why you should join us?</h3>
+      <p class="description">This year’s convention is very special in many ways, as we are working to ensure 
+        everybody enjoys the best camping experiences in the area of spiritual encounters, accommodation, 
+        transportation, security, water system, electricity, welfare, program spacing and hubs transmission system. 
+        The God of Supernatural Encounter has been faithful in helping us on all the projects we prayerfully 
+        embarked upon. If I were you, I would be there.
+      </p>
+      <div class="timestamp">
+        <h3>2023.05.17(WED) ~ 21(SUN)</h3>
+        <p>@Foursquare Gospel Church, Ongwediva, Namibia</p>
+      </div>
+    </div>
+  </section>
+
+  <section class="program">
+
+    <div class="set-margin">
+      <div class="d-flex">
+        <h4>Main Program</h4>
+      </div>
+      <div class="cards">
```

Comment:
- [ ] I kindly suggest that you add one more individual card to make your cards up to 5 as seen in the template design given.

| ------------- | ------------- |
| ------------- | ------------- |
| Your Design | <img width="831" alt="image" src="https://user-images.githubusercontent.com/52934128/217907353-cca88c69-dfd9-42c4-92a5-060f45f84aca.png">  |
| Template Design  | <img width="628" alt="image" src="https://user-images.githubusercontent.com/52934128/217907268-8a62c63a-9a3f-416a-af83-9d6a67781e74.png">  |